scripts/uploadReport.sh

```
@@ -14,6 +14,9 @@ USER=$1
 PASS=$2
 TYPE=$4
 
+set -x
+set -e
```

Is this needed? I "fear" that this might expose password/sensitive data

---

I removed `set -x` which enables debug trace.

`set -e` causes script to fail fast on first non-zero exit code, preventing Jenkins from running `curl` when previous commands fail for whatever reason (we had multiple cases of `curl` trying to upload something for hours).

---

:+1: for -e